src/app/course/course-routing.module.ts

```diff
@@ -61,5 +61,6 @@ const routes = [
     imports: [RouterModule.forChild(routes)],
     exports: [RouterModule]
 })
+
```

Comment:
These changes are here because you moved the component. Please change it back to what it originally was.